### PR TITLE
Regenerated SDK basing on specs updates; Added tests

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
## Description

---
PR: https://github.com/Azure/azure-rest-api-specs/pull/1276

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by DeepakRajendranMsft — APPROVED** [review]
looks like we are using a new version of auto rest code gen which is changing almost all files

**Comment by DeepakRajendranMsft** [comment]
@cormacpayne  please review this. we need these for PS changes

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
All new tests should be recorded using latest nugets packages (e.g. resource manager, storage etc)
Definitely newer resource manager >=1.6.0

**Comment by EvgenyAgafonchikov** [comment]
updated packages, re-recorded tests. @shahabhijeet please have another look

File: src/SDKs/Network/Network.Tests/Network.Tests.csproj [hunk 1]
```diff
@@ -35,4 +35,12 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  
+  <ItemGroup>
```
Comment (shahabhijeet) [inline src/SDKs/Network/Network.Tests/Network.Tests.csproj]: @EvgenyAgafonchikov you do not need this, it adds warning as your test project implicitly reference these, else no tests would have run on the first place.
Remove these redundant entries

**Comment by shahabhijeet** [comment]
@EvgenyAgafonchikov can you point me out the file that actually uses the new ResourceManagementClient?

**Review by shahabhijeet — APPROVED** [review]

File: src/SDKs/Network/Network.Tests/Network.Tests.csproj (hunk 1 above)
Comment (shahabhijeet) [inline src/SDKs/Network/Network.Tests/Network.Tests.csproj]: @EvgenyAgafonchikov please remove these redundant entries

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**